services/QuillLMS/spec/workers/teacher_activity_feed_batch_refill_worker_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe TeacherActivityFeedBatchRefillWorker, type: :worker do
+  let(:worker) { described_class.new }
+  let!(:teacher_in_window1) { create(:teacher, last_sign_in: '2020-01-05')}
+  let!(:teacher_in_window2) { create(:teacher, last_sign_in: '2020-01-06')}
+  let!(:teacher_outside) { create(:teacher, last_sign_in: '2020-02-10')}
+
+  it 'should queue jobs for teachers with last sign-in within start/end' do
+    expect(TeacherActivityFeedRefillWorker).to receive(:perform_in).with(0, teacher_in_window1.id)
+    expect(TeacherActivityFeedRefillWorker).to receive(:perform_in).with(7, teacher_in_window2.id)
+
+    worker.perform('2020-01-01', '2020-01-31', 7)
```

Comment:
It took me a few seconds figure out that the `7` in the expect was being provided as part of the `perform` call.  (I had the default of "3" in my head.). Seems like one of those places where it might be worth naming the variable.  "SPECIFIED_DELAY" or something?  Not a huge deal, just figured I'd mention it.

Reply:
Yeah, I'll update the test to make it clear. This is also a place where it would be nice if [sidekiq support keyword arguments,](https://github.com/mperham/sidekiq/issues/2372) but it does not.